src/trusted/service_runtime/sel_main.c

```diff
@@ -688,16 +733,19 @@ int NaClSelLdrMain(int argc, char **argv) {
       errcode = NaClAppPrepareToLaunch(nap);
       if (LOAD_OK != errcode) {
         nap->module_load_status = errcode;
-        fprintf(stderr, "NaClAppPrepareToLaunch returned %d", errcode);
+        NaClLog(1, "NaClAppPrepareToLaunch returned %d", errcode);
       }
-      NaClPerfCounterMark(&time_all_main, "AppPrepLaunch");
-      NaClPerfCounterIntervalLast(&time_all_main);
     }
 
     /* Give debuggers a well known point at which xlate_base is known.  */
     NaClGdbHook(&state);
   }
 
+#if NACL_OSX
+# define _HOST_OSX 1
+#else
+# define _HOST_OSX 0
+#endif
```

Comment:
is there a reason you can't just use `NACL_OSX` rather than `_HOST_OSX`?

Reply:
Yes, it messes up linters in conditionals like https://github.com/Lind-Project/native_client/blob/lind_fork_changes/src/trusted/service_runtime/sel_main.c#L757

Since the conditional becomes `if (a &&)` which is invalid syntax. 

Granted, it's not super important, though. Is it better removed in your opinion?

Reply:
Oh no, I was just curious. This looks like a valid reason imho